### ISSUE TITLE
Add docs for CustomerAddress service

### ISSUE THREAD

**Issue by nozzlegear** [nozzlegear/ShopifySharp#197]

**Comment by jeremysimmons** [comment]
docs go in the README.md file in the root of the repository, right?

**Comment by nozzlegear** [comment]
Yep, for now they go in there. I'm planning on a better solution so it isn't all crammed into the same document but that's a few weeks away still.
